vignettes/logrx.Rmd

```diff
@@ -25,22 +25,34 @@ an R script which enables traceability and reproducibility of the executed code.
 # Log attributes
 The following attributes are recorded in the log:
 
-* `metadata`: The logrx `package` information
-* `user and file information`: The user that generates the log, file name and path of the R script
+* `metadata`: logrx `package` information
+   + Version of the package
+   + Type of build
+   + Link to the GitHub repository
+* `user and file information`
+   + User that generated the log
+   + Name and path of the script for which the log was generated
 * `hash_sum`:  A unique sha1 is created for the log file
```

Comment:
```suggestion
* `hash_sum`:  A unique hashsum is created for the log file
```

Reply:
The hash_sum should go under the `user and file info` section.  We also need a new log rendered for the screenshot that has the `File HashSum:` section in it